### DISCUSSION TITLE
How to close app?

### DISCUSSION THREAD

**Discussion by RGPATCHI** [ansys/pymechanical#475] [Q&A] [answered]
From bug reports, I see that from 24R1, we will have Application.Exit() to close Embedded Mechanical session. 
What is the procedure for closing it in 23R2 or earlier?

Do we need to use Task manager? or is there any better way?

**Comment by pmaroneh** [comment]
@koubaa , what is the best way here? Many thanks

**Comment by klmcadams** [comment, marked as answer]
Hi @RGPATCHI @pmaroneh, I believe you can still call app.exit() for 23R2. If the version is < 241, we use ExtAPI.Application.Close(). Otherwise, we use ExtAPI.Application.Exit()
